eip_6492/eip_6492.go

```diff
@@ -0,0 +1,223 @@
+package eip6492
```

Comment:
can you rename the folder from "eip_6492" to just "eip6492" please? go convention is for folders to use the same name as their packages